activerecord/lib/active_record.rb

```diff
@@ -301,7 +301,7 @@ def self.global_thread_pool_async_query_executor # :nodoc:
   # with the global thread pool async query executor.
   def self.global_executor_concurrency=(global_executor_concurrency)
     if self.async_query_executor.nil? || self.async_query_executor == :multi_thread_pool
-      raise ArgumentError, "`global_executor_concurrency` cannot be set when using the executor is nil or set to multi_thead_pool. For multiple thread pools, please set the concurrency in your database configuration."
+      raise ArgumentError, "`global_executor_concurrency` cannot be set when using the executor is nil or set to multi_thread_pool. For multiple thread pools, please set the concurrency in your database configuration."
```

Comment:
I think `using` shouldn't be used here. Also maybe use backticks for `:multi_thread_pool` as well?
```suggestion
      raise ArgumentError, "`global_executor_concurrency` cannot be set when the executor is nil or set to `:multi_thread_pool`. For multiple thread pools, please set the concurrency in your database configuration."
```

Reply:
Yes, I agree. Fixed it